hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneClusterImpl.java

```diff
@@ -340,19 +340,20 @@ public void stop() {
       ozoneManager.join();
     }
 
+    if (!hddsDatanodes.isEmpty()) {
+      LOG.info("Shutting the HddsDatanodes");
+      hddsDatanodes.parallelStream()
+          .forEach(dn -> {
+            dn.stop();
```

Comment:
shall we wrap them in try catch to cleanup all of them silently.

Reply:
Thanks for the review Ajay, I didnot understand this comment completely. Can you please elaborate. ?